Documentation/O3DEMPSProjectExportTesting.md

```diff
@@ -0,0 +1,58 @@
+# O3DE MPS Project Export Testing Instructions
+
+## Notes
+
+* The general O3DE project-centric export script is still in development. For now the project-centric instructions will be calibrated using the MPS Gamelift Export Script. Once the general script is merged, the instructions will be updated.
+* This instruction set assumes the host platform is Windows. Instructions for Linux coming shortly.
+
+## Fresh Install and Setup
+1. Clone the development branch of O3DE:
+```bash
+cd \path\to\workspace
+git clone https://github.com/o3de/o3de.git # this should already be in development branch
```

Comment:
The default branch might be a local setting.
It would be better to be explicit here. With git you can actually clone into a specific branch.

```
git clone -b <branch> <remote_repo>
```